README.md

```diff
@@ -31,3 +31,32 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## My solution:
+
+$T(n)=1$ If $n \leq 1$, $3T(n/3)+n^5$ If $n>1$
+<br>
+<br>
+$3T(\frac{n}{3})+n^5$
+<br>
+<br>
+$9T(\frac{n}{9})+n^5+\frac{n^5}{3^4}$
+<br>
+<br>
+$27T(\frac{n}{27})+n^5+\frac{n^5}{3^4}+\frac{n^5}{9^4}$
+<br>
+<br>
+### Relation
+$i$: Iterations
+<br>
+$3^iT(\frac{n}{3^i})+{n^5\sum_{n=1}}^i(\frac{1}{3})^{4n}$
```

Comment:
Not sure about the sum here. Can you explain how you got this? It looks quite different to what we did in class.

Reply:
I appreciate your feedback. Is my logic clearer now?